b2b_ecommerce/views_test.py

```diff
@@ -168,6 +168,27 @@ def test_create_order_with_invalid_code(client):
     assert resp.json() == {"errors": ["Invalid coupon code"]}
 
 
+def test_create_order_with_invalid_email(client):
```

Comment:
Can you make this test parametrized? e.g. you can test this on multiple values e.g. `"", "something", "something@", "@something"`

Reply:
added